bool MCBsSB_5V
bool MCBs_12V
bool videotransmitter
bool fan
bool laser
bool gps
bool arm_5V
bool arm_12V
bool pc2
bool camera
